torch/_meta_registrations.py

```diff
@@ -584,6 +584,13 @@ def cholesky(self: Tensor, upper: bool = False) -> Tensor:
     return cloneBatchedColumnMajor(self)
 
 
+@register_meta(aten.cholesky_inverse)
+@out_wrapper()
+def cholesky_inverse(self: Tensor, upper: bool = False) -> Tensor:
+    squareCheckInputs(self, "cholesky_inverse")
+    return cloneBatchedColumnMajor(self)
+
+
```

Comment:
cholesky_inverse_out in aten/src/ATen/native/BatchLinearAlgebra.cpp does a bunch of things related to copying, this is not exactly 1-to-1, but seems to work.